### DISCUSSION TITLE
Enitity update can only update?

### DISCUSSION THREAD

**Discussion by JieZeng1993** [SeaQL/sea-orm#1887] [Ideas]
Enitity Insert execute, just do  Insert.
ActiveModel Insert execute, do Insert and select.

why Enitity update and ActiveModel update  do Update and select.

can seaorm provide a method only do update?

Now, I do update only by update_many

**Comment by tyt2y3** [comment]
Umm.. what's insufficient about update_many, other than that the name is not exactly descriptive?
